File: 프로그래머스/unrated/120806. 두 수의 나눗셈/두 수의 나눗셈.java
/*
class Solution {
    public int solution(int num1, int num2) {
        int answer = (1000 * num1 / num2);
        return answer;
    }
}*/

class Solution {
    public int solution(int num1, int num2) {
        double answer = (double) num1 / (double) num2;
        return (int) (answer * 1000);
    }
}

File: 프로그래머스/0/120806. 두 수의 나눗셈/두 수의 나눗셈.cs
using System;

public class Solution {
    public int solution(int num1, int num2) {
        int answer = calc(num1, num2);
        return answer;
    }
    
    private int calc(int num1, int num2){
        
        double cal1 = (double)num1 / num2;
        cal1 *= 1000;
        return (int)Math.Truncate(cal1);
    }
}